mmcv/runner/checkpoint.py

```diff
@@ -106,14 +106,43 @@ def load(module, prefix=''):
 
 
 def get_torchvision_models():
-    model_urls = dict()
-    for _, name, ispkg in pkgutil.walk_packages(torchvision.models.__path__):
-        if ispkg:
-            continue
-        _zoo = import_module(f'torchvision.models.{name}')
-        if hasattr(_zoo, 'model_urls'):
-            _urls = getattr(_zoo, 'model_urls')
-            model_urls.update(_urls)
+    if digit_version(torchvision.__version__) <= digit_version('0.12.0'):
+        model_urls = dict()
+        # When the version of torchvision is lower than 0.13, the model url is
+        # not declared in `torchvision.model.__init__.py`, so we need to
+        # iterate through `torchvision.models.__path__` to get the url for each
+        # model.
+        for _, name, ispkg in pkgutil.walk_packages(
+                torchvision.models.__path__):
+            if ispkg:
+                continue
+            _zoo = import_module(f'torchvision.models.{name}')
+            if hasattr(_zoo, 'model_urls'):
+                _urls = getattr(_zoo, 'model_urls')
+                model_urls.update(_urls)
+    else:
+        # Since torchvision bumps to 0.13, the weight loading logic, model keys
+        # and model urls have been changed. We will load the old version urls
+        # in mmcv/model_zoo/torchvision_0.12.json to prevent from BC
+        # Breaking. If your version of torchvision is higher than 0.13.0,
+        # new urls will be added to `model_urls` additionally. You can get the
+        # newest torchvision model by resnet50.imagenet1k_v1.
+        json_path = osp.join(mmcv.__path__[0],
+                             'model_zoo/torchvision_0.12.json')
+        model_urls = mmcv.load(json_path)
+        for cls_name, cls in torchvision.models.__dict__.items():
+            if (not cls_name.endswith('_Weights')
+                    or not hasattr(cls, 'DEFAULT')):
```

Comment:
Why do we need to check whether clas has `DEFAULT`?

Reply:
some model weight classes, such as `MNASNet0_75_Weights` does not have any urls in torchvision 0.13.0 and cannot be iterated. Here we simply check `DEFAULT` attribute to ensure the class is not empty.  I've supplemented this comment in code.